doc/releases/release-notes-4.0.rst

```diff
@@ -1220,14 +1224,8 @@ Libraries / Subsystems
     non-volatile storage technologies. It supports classical on-chip NOR flash as well as
     new technologies like RRAM and MRAM that do not require a separate erase operation at all.
 
-* Task Watchdog
-
 * POSIX API
```

Comment:
note that POSIX API was intentionally kept to not clash with https://github.com/zephyrproject-rtos/zephyr/pull/81442